### PR TITLE
[PVM] Add the functionality of multisig alias definition removal in MultisigAliasTx

### PR DESCRIPTION
**Pull request by evlekht**
## Why this should be merged
This PR adds the functionality of removing a multisig alias definition. Alias cannot be removed if its consortium member or role admin.
## How this works
If MultisigAliasTx contains empty multisig alias definition, but non-empty alias ID, than its removal.
To clarify: empty alias definition means that owners has no address and threshold is 0.
## How this was tested
By existing unit test with newly added test cases.
## Co-authors:
See original PR.
## Additional references
Original PR based on cortina-15 dev
https://github.com/chain4travel/caminogo/pull/356

### CODE REVIEW

File: vms/components/multisig/camino_multisig.go [hunk 1]
```diff
@@ -16,10 +17,23 @@ import (
...
 
+var (
+	_ verify.State = (*Alias)(nil)
+
+	errMemoIsTooBig = errors.New("msig alias memo is too big")
+	errEmptyAlias   = errors.New("alias id and alias owners cannot be empty both at the same time")
+)
+
+type Owners interface {
+	verify.State
```
Comment (mo-c4t) [inline vms/components/multisig/camino_multisig.go:28]: snow.ContextInitializable is missing from the original PR

File: vms/components/multisig/camino_multisig.go [hunk 2]
```diff
@@ -16,10 +17,23 @@ import (
 // MaxMemoSize is the maximum number of bytes in the memo field
 const MaxMemoSize = 256
 
+var (
+	_ verify.State = (*Alias)(nil)
```
Comment (mo-c4t) [inline vms/components/multisig/camino_multisig.go:21]: _ snow.ContextInitializable = (*Alias)(nil) missing from original pr

File: vms/components/multisig/camino_multisig.go [hunk 3]
```diff
@@ -16,10 +17,23 @@ import (
...
+)
+
+type Owners interface {
+	verify.State
+
+	IsZero() bool
+}
+
 type Alias struct {
 	ID     ids.ShortID         `serialize:"true" json:"id"`
```
Comment (mo-c4t) [inline vms/components/multisig/camino_multisig.go:34]: verify.IsNotState `json:"-"` is missing from original PR

File: vms/components/multisig/camino_multisig.go (hunk 1 above)
Comment (evlekht) [reply vms/components/multisig/camino_multisig.go:28]: Yes, its part of old verify.State interface

File: vms/components/multisig/camino_multisig.go (hunk 2 above)
Comment (evlekht) [reply vms/components/multisig/camino_multisig.go:21]: its part of old verify.State interface

File: vms/components/multisig/camino_multisig.go (hunk 3 above)
Comment (evlekht) [reply vms/components/multisig/camino_multisig.go:34]: thats not needed cause we're using old verify.State interface, cause its not restructured yet

**Review by mo-c4t — DISMISSED** [review]

**Review by mo-c4t — APPROVED** [review]

**Pull request merged**